### ISSUE TITLE
iphone notifications for automatic approvals need to be turned off

### ISSUE THREAD

**Issue by oregonpillow** [Ombi-app/Ombi#4209]
In the iphone app, all my users are set for auto-approvals. yet, i get notifications saying "<user> has requested <something>, sign in to ombi to approve the request"

even if i click the notification (which is pointless) it will re-direct me to the request to show me that it's already approved (like i setup).

Please turn off notifications for approvals when it's set to auto approve. it was so annoying i had to turn off all notifications for the app in ios since there are no notification settings within the app itself.

Now i'm greeted with an error message that notifications are turned off when i open the app....will delete app and re-install once this issue fixed.

in summary:
- turn off approval notifications for auto approvals
- allow notification settings within app

**Comment by tidusjar** [comment]
You can turn off these notifications from the mobile notification settings page in ombi

**Comment by tidusjar** [comment]
For example ![Screenshot_20210526-192354_Chrome.jpg](https://user-images.githubusercontent.com/6642220/119711900-f19d9280-be57-11eb-88e5-c5fa574a0307.jpg)

**Comment by oregonpillow** [comment]
@tidusjar thanks, however on ios app there is no such menu on ombi.

Within the latest ios app, going to settings has "User Settings" and "Admin" only. And none of these have notifications within them.

**Comment by tidusjar** [comment]
I'm referring to the ombi website

**Comment by oregonpillow** [comment]
@tidusjar  Found it now :) Thanks!
